Sources/App/Views/PackageController/Builds/BuildShow+View.swift

```diff
@@ -87,6 +89,13 @@ enum BuildShow {
                             .strong(.text(model.reference))
                         )
                     },
+                    .unwrap(model.buildInfo.buildDetails) {
+                        .group(
+                            .text(" (\($0.commitHash.prefix(6)))"),
+                            .text(" on "),
+                            .strong(.text(DateFormatter.displayFormatter.string(from: $0.buildDate) + " UTC"))
```

Comment:
We do it in a separate PR if you like, but we should also add build duration here. People have no way of telling whether they are near our timeout at the moment.

Reply:
We should add that but more importantly I'd like to display the Swift 6 error count per build here as well. At that point that shouldn't just be a run-on sentence but rather something like a list

```
Build date:
Build duration:
Swift 6 errors:
```

Reply:
👍

If you want to open a PR that brings the Swift 6 error data through I can format it into the page in a way that makes sense.